lib/board.rb

```diff
@@ -41,4 +41,83 @@ def update_grid(location, mark)
     end
   end
 
+  def full_board?(board_grid)
```

Comment:
Consider naming this just `full?`. It's a bit redundant to call it `full_board?` since this method is on the board itself.

Reply:
Yes, makes sense. Done!